alembic/versions/51e4adb4b60_add_watch_list.py

```diff
@@ -28,6 +28,15 @@ def upgrade():
         sa.Column('action', sa.UnicodeText(), nullable=False),
     )
 
```

Comment:
Hard-coding a value here probably isn't the best way to do this.  I understand that you want to avoid needing to import the value from the codebase, since that value might not exist across all versions for all time.

A more past-and-future-proof way of doing this in the future would be to extract the snippet of code we use to compute constant values into its own function and then call that.  The snippet is in ``constants.py`` and is as follows:

```python
int(sha512(name.upper().encode()).hexdigest()[:7], 16)
```

which we could extract into a function like

```python
def constant_value(name):
    return int(sha512(name.upper().encode()).hexdigest()[:7], 16)
```

and then instead of saying ``default=252034462`` we could say ``default=constant_value('shift_unworked')``, which is both less brittle and much easier for someone coming to this file later to read.

What you have here is fine, of course, this is more of a note for how we should probably do this in the future.